samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMatching.jsp

```diff
@@ -38,20 +38,10 @@ should be included in file importing DeliveryMessages
   </t:dataList>
 
   <h:dataTable value="#{question.matchingArray}" var="matching">
-    <h:column rendered="#{delivery.feedback eq 'true' &&
-       delivery.feedbackComponent.showCorrectResponse && !delivery.noFeedback=='true'}">
-      <h:panelGroup id="image"
-        rendered="#{matching.isCorrect}"
-        styleClass="si si-check-lg" >
-      </h:panelGroup>
-      <h:panelGroup id="ximage"
-        rendered="#{!matching.isCorrect}"
-        styleClass="si si-remove feedBackCross" >
-      </h:panelGroup>
-      <h:graphicImage id="image2"
-        width="16" height="16"
-        alt="#{deliveryMessages.alt_incorrect}" url="/images/delivery/spacer.gif">
-      </h:graphicImage>
+    <h:column rendered="#{delivery.feedback eq 'true' && delivery.feedbackComponent.showCorrectResponse && !delivery.noFeedback=='true'}">
+      <h:panelGroup id="image" rendered="#{matching.isCorrect eq 'true'}" styleClass="si si-check-lg" > </h:panelGroup>
+      <h:panelGroup id="ximage" rendered="#{not empty matching.isCorrect && matching.isCorrect eq 'false'}" styleClass="si si-remove feedBackCross"> </h:panelGroup>
+      <h:graphicImage id="image2" width="16" height="16" alt="#{deliveryMessages.alt_incorrect}" url="/images/delivery/spacer.gif"> </h:graphicImage>
```

Comment:
Using `spacer.gif `seems like it's not really needed and a `margin-right` or other CSS could be used now on the other panelGroup's. But since this is used so many places in Samigo it's probably a separate cleanup issue for this in general.

Reply:
yes would be a nice cleanup to get rid of all images in samigo!!